src/lib/eliom_notif.server.ml

```diff
@@ -188,3 +217,29 @@ module Make (A : S) = struct
     Lwt.return @@ Notif_hashtbl.iter f I.tbl
 
 end
+
+module type SIMPLE = sig
+  type identity
+  type key
+  type notification
+  val equal_key                  : key -> key -> bool
+  val equal_identity             : identity -> identity -> bool
+  val get_identity               : unit -> identity Lwt.t
+  val max_resource               : int
+  val max_identity_per_resource  : int
+end
+
+module Simple(A : SIMPLE) = Make
+  (struct
```

Comment:
Same as above for `SIMPLE` (`ARG_SIMPLE`?). Maybe also rename `Simple` to something that sounds more like a functor, e.g., `Make_simple`?